Documentation/Index.rst

```diff
@@ -15,146 +15,115 @@ installed.
 
 ----
 
-.. container:: row m-0 p-0
+..  card-grid::
+    :columns: 1
+    :columns-md: 2
+    :gap: 4
+    :class: pb-4
+    :card-height: 100
 
-   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+    ..  card:: Concepts
 
-      .. container:: card px-0 h-100
+        Written for new users, this chapter introduces some of TYPO3's core
+        concepts including the backend, TYPO3's administration interface.
 
-         .. rst-class:: card-header h3
+        ..  card-footer:: :ref:`Learn about the basic concepts <Concepts>`
+            :button-style: btn btn-secondary stretched-link
 
-            .. rubric:: :ref:`Concepts <Concepts>`
+    ..  card:: System requirements
 
-         .. container:: card-body
+        System requirements for the host operation system, including its web
+        server and database and how they should be configured prior to
+        installation.
 
-            Written for new users, this chapter introduces some of TYPO3s core concepts including the backend, TYPO3s administration interface.
+        ..  card-footer:: :ref:`Go to the System Requirements <System-Requirements>`
+            :button-style: btn btn-secondary stretched-link
 
-   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+    ..  card:: Installation
 
-      .. container:: card px-0 h-100
+        The installation chapter provides detailed instructions about how to
+        install TYPO3, it also contains information about how to deploy TYPO3
+        to a production environment.
 
-         .. rst-class:: card-header h3
+        ..  card-footer:: :ref:`Install TYPO3 <Install>`
```

Comment:
```suggestion
        ..  card-footer:: :ref:`Perform a TYPO3 Installation <Install>`
```

I'd like to avoid a button that sounds like clicking on it would Install TYPO3 for you. Since the button labels are all active verbs at the beginning, we need another one to not create this expectation.